omnibus/config/software/libbcc_compat.patch

```diff
@@ -109,9 +112,19 @@ index 0000000..c01fbe2
 +  return __ ## func ## _prior_glibc(x);                         \
 +}
 +
++#define define_fmemopen_wrapper_for(func)                                      \
++FILE *__ ## func ## _prior_glibc(void *buf, size_t size, const char *mode);    \
++                                                                               \
++asm(".symver __" #func "_prior_glibc, " #func "@" GLIBC_VERS);                 \
++                                                                               \
++FILE *__wrap_## func (void *buf, size_t size, const char *mode) {           \
++ return __ ## func ## _prior_glibc(buf, size, mode);                           \
++}                                                                              \
++
 +define_wrapper1_for(exp)
 +define_wrapper1_for(log)
 +define_wrapper2_for(pow)
 +define_wrapper1_for(exp2)
 +define_wrapper1_for(log2)
 +define_wrapper3_for(log2f)
++define_fmemopen_wrapper_for(fmemopen)
```

Comment:
Do we need to pass in `fmemopen` here since it is in the macro name?

Reply:
Yes, the macro takes an argument